frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsHours/FeatureMetricsHours.tsx

```diff
@@ -62,7 +60,24 @@ const hourOptions: { key: `${number}`; label: string }[] = [
         label: 'Last 24 hours',
     },
     {
-        key: `${FEATURE_METRIC_HOURS_BACK_MAX}`,
-        label: `Last ${FEATURE_METRIC_HOURS_BACK_MAX} hours`,
+        key: '48',
+        label: 'Last 48 hours',
+    },
+];
+
+const now = new Date();
+
+const daysOptions: { key: `${number}`; label: string }[] = [
+    {
+        key: `${differenceInHours(now, subWeeks(now, 1))}`,
+        label: 'Last week',
+    },
+    {
+        key: `${differenceInHours(now, subMonths(now, 1))}`,
```

Comment:
It differs from setting it to 30 days. The date-fns library is adjusting the month part of the date while keeping other components (like the day, hour, minute, second) the same